debug/captions.html

```diff
@@ -253,87 +271,124 @@ <h3>footer</h3>
 
     m2ts.addEventListener('change', function() {
       var reader = new FileReader();
+      var remuxedSegments = [];
+      var parsedCaptions = [];
+      var remuxedInitSegment = null;
+      var remuxedBytesLength = 0;
+      var createInitSegment = false;
+      var bytes;
+
+      var combined = document.querySelector('#combined-output').checked;
+      var outputType = document.querySelector('input[name="output"]:checked').value;
+      var resetTransmuxer = $('#reset-tranmsuxer').checked;
+      var partial = $('#partial-transmuxer').checked;
 
       // do nothing if no file was chosen
       if (!this.files[0]) {
         return;
       }
 
-      reader.addEventListener('loadend', function() {
+      var onData = function(event) {
+        if (event.type === outputType) {
```

Comment:
Can we get other types?

Reply:
You can get data events from the AudioSegmentStream if you don't remux